src/microbe_stage/systems/StrainSystem.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Components;
+using DefaultEcs;
+using DefaultEcs.System;
+using DefaultEcs.Threading;
+using Godot;
+using World = DefaultEcs.World;
+
+[With(typeof(OrganelleContainer))]
+[With(typeof(StrainAffected))]
+[ReadsComponent(typeof(OrganelleContainer))]
+[ReadsComponent(typeof(MicrobeControl))]
+public sealed class StrainSystem : AEntitySetSystem<float>
+{
+    public StrainSystem(World world, IParallelRunner runner) : base(world, runner)
```

Comment:
```suggestion
    public StrainSystem(World world, IParallelRunner runner) : base(world, runner, Constants.SYSTEM_EXTREME_ENTITIES_PER_THREAD)
```

I'd add that to prepare this system to be multithreaded in the future.

Reply:
It tells me I need to declare content when I add this, I'm kinda confused